samples/wifi/sta/README.rst

```diff
@@ -87,6 +87,8 @@ You can change the following default static configuration in the :file:`prj.conf
   CONFIG_NET_CONFIG_MY_IPV4_NETMASK="255.255.255.0"
   CONFIG_NET_CONFIG_MY_IPV4_GW="192.168.1.1"
 
+.. _wifi_station_sample_building_and_running:
+
```

Comment:
Line 235 below: 
You can use the ~~at~~ :file:`overlay-zperf.conf` file to run the performance test.